apps/design-system/src/subjects/views/pipeline-edit/pipeline-execution.tsx

```diff
@@ -10,13 +10,15 @@ const PipelineExecution = () => {
   return (
     <div className="relative flex h-screen grow">
       <PipelineStudioNodeContextProvider
-        onSelectIntention={() => {}}
-        onAddIntention={() => {}}
-        onEditIntention={() => {}}
-        onDeleteIntention={() => {}}
-        onRevealInYaml={() => {}}
+        selectedPath={''}
+        onAddIntention={() => undefined}
+        onDeleteIntention={() => undefined}
+        onEditIntention={() => undefined}
+        onRevealInYaml={() => undefined}
+        onSelectIntention={() => undefined}
```

Comment:
Sending new function instance will make this component er-render everytime. Instead we can pass `noop` from lodash or our own.